git hub actions is giving me an error

![image](https://user-images.githubusercontent.com/74893977/169697627-aae395de-c0e7-493d-bd3a-ddd4fae4d322.png)

{"code":"ENOENT","errno":-2,"level":"error","message":"ENOENT: no such file or directory, stat 'plot.png'","path":"plot.png","stack":"Error: ENOENT: no such file or directory, stat 'plot.png'","syscall":"stat"}
Error: Process completed with exit code 1.

I'm having this error when I'm pushing the repo I forked, I added the code you guys gave me into cml.yml. And when git hub actions is running the trani.py part of the cml.yml file the above error is occurring. Please help, I really want to start using this cml thing.

--- a/train.py
+++ b/train.py
@@ -25,4 +25,4 @@ with open("metrics.txt", "w") as outfile:
 disp = ConfusionMatrixDisplay.from_estimator(
     clf, X_test, y_test, normalize="true", cmap=plt.cm.Blues
 )
-plt.savefig("confusion_matrix.png")
+plt.savefig("plot.png")

